health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/FacilityCatchmentMappnig.js

```
@@ -0,0 +1,22 @@
+
+const FacilityCatchmentMapping = ({}) => {
+    const { t } = useTranslation();
+    const config = facilityMappingConfig();
+
+    return (
+        <React.Fragment>
+          <Header>{t("MICROPLAN_ASSIGN_CATCHMENT_VILLAGES")}</Header>
+          <div className="inbox-search-wrapper">
+            <InboxSearchComposer
+              configs={config}
+            ></InboxSearchComposer>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)